support-frontend/assets/pages/subscriptions-landing/components/featuredProducts.jsx

```diff
@@ -168,6 +169,9 @@ const getProduct = (subsLinks: SubsUrls, countryGroupId: CountryGroupId): ?Produ
     case 'GuardianWeekly':
       return products.GuardianWeekly;
     default:
+      if (promoteGuardianWeekly) {
```

Comment:
This means we will no longer be promoting the paper in the UK, is that what we want?

Reply:
I also wondered this. I've asked Chris about it.